client/src/components/layout/SignOutButton.js

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from "react";
+import { Redirect } from "react-router-dom";
+
+const SignOutButton = () => {
```

Comment:
is there a reason the location of this file changed from the `authentication` folder?